darwin/future/tests/meta/objects/test_itemmeta.py

```diff
@@ -44,3 +44,28 @@ def test_delete(item: Item) -> None:
             json={},
         )
         item.delete()
+
+
+def test_move_to_folder(item: Item) -> None:
+    with responses.RequestsMock() as rsps:
```

Comment:
Consider testing action result, and also adding sad path.

Reply:
Good shout on the sad path, have added - On the subject of testing the action result, how would you approach this?

In this case, `move_to_folder()` doesn't return anything to check against, and the request is mocked. What can we check against to verify that the action had the intended effect?